lambdas/handlers/document_manifest_by_nhs_number_handler.py

```diff
@@ -33,68 +29,34 @@
 @override_error_check
 def lambda_handler(event, context):
```

Comment:
@ensure_environment_variables might be good to add

Reply:
It's already in if you expand the code up it's just hidden from view

Reply:
bwahaha I pulled it and was looking but missed it anyways! What a n00b